services/ui-src/src/routes/Intro/Intro.tsx

```diff
@@ -1,29 +1,138 @@
 // components
-import { Box, Heading, Text } from "@chakra-ui/react";
-import { ReportPage } from "components";
+import { Box, Button, Image, Heading, Text, Flex } from "@chakra-ui/react";
+import { BasicPage, IntroSection, SpreadsheetWidget } from "components";
+import { useNavigate } from "react-router-dom";
+// utils
+import { makeMediaQueryClasses } from "utils";
 import verbiage from "verbiage/pages/mcpar/mcpar-intro";
+// assets
+import nextIcon from "assets/icons/icon_next.png";
 
 export const Intro = () => {
-  const { intro } = verbiage;
+  const { intro, body, pageLink } = verbiage;
+  const mqClasses = makeMediaQueryClasses();
+  const navigate = useNavigate();
+
+  const [section1, section2, section3] = body.sections;
+
   return (
-    <ReportPage data-testid="intro-view">
+    <BasicPage sx={sx.layout} data-testid="intro-view">
       <Box sx={sx.leadTextBox}>
         <Heading as="h1" sx={sx.headerText}>
           {intro.header}
         </Heading>
-        <Text>{intro.body}</Text>
       </Box>
-    </ReportPage>
+      <div>
+        <IntroSection index={1} content={section1} key={`section-1`}>
```

Comment:
Since you're already passing in all the section content, I'd suggest including the `index` here (name should be changed as noted above) in the content that's stored in the json as like `sectionNumber`. Also I don't think the `key` prop is needed anymore.
```suggestion
        <IntroSection content={section1}>
```